### PR TITLE
Sql injection param protection

### LINKED ISSUE #76: C# Protect CosmosDBService from SQL Injection Attacks
## Description
I believe that CosmosDBService is currently susceptible to SQL injection attacks because SQL strings are being built with simple string concatenation.

## Reproduce
If we were to call one of CosmosDBService's methods such as queryActors using a malicious query string parameter such as an actor name of "Joe Smith; DELETE * FROM actors;" then we have an attack vector.

## Additional Context 
- _(If applicable: Add any other context about the problem here; for example: doc changes, screenshots, logs, links, etc)_

References https://github.com/retaildevcrews/helium/issues/334

### PR DESCRIPTION
**Pull request by mag16**
## Type of PR
PR to switch to CosmosDB native support for parameterized queries to prevent SQL injection attacks.
- [ ] Documentation changes
- [X] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Parameterize queries to protect against SQL injection attacks.
## Issues Closed or Referenced

- Closes #76

### CODE REVIEW

File: src/app/GlobalSuppressions.cs [hunk 1]
```diff
@@ -20,3 +20,4 @@
 
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("IDE", "IDE0051:Private member GetXmlCommentsPath is unused", Justification = "can be used if configured", Scope = "type", Target = "Helium.Startup")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1054:Uri parameters should not be strings", Justification = "KeyVault SDK expects string", Scope = "type", Target = "Helium.App")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "<Pending>", Scope = "member", Target = "~M:Helium.DataAccessLayer.DAL.GetActorsAsync(System.String,System.Int32,System.Int32)~System.Threading.Tasks.Task{System.Collections.Generic.IEnumerable{Helium.Model.Actor}}")]
```
Comment (bartr) [inline src/app/GlobalSuppressions.cs]: why is this suppressed?

No justification

File: src/app/DataAccessLayer/dalMovies.cs [hunk 2]
```diff
@@ -154,5 +175,7 @@ public async Task<List<string>> GetFeaturedMovieListAsync()
 
             return list;
         }
+
     }
-}
+
+}
```
Comment (bartr) [inline src/app/DataAccessLayer/dalMovies.cs:182]: linter issue - missing final LF - need to run code cleanup as there are other linter errors

File: src/app/DataAccessLayer/dalMain.cs [hunk 3]
```diff
@@ -149,6 +149,31 @@ public static string GetPartitionKey(string id)
...
+                foreach (var doc in await query.ReadNextAsync().ConfigureAwait(false))
+                {
+                    results.Add(doc);
+                }
+            }
+
+            return results;
+        }
+
+        // TODO: Will remove original sql query once all callers are converted
```
Comment (bartr) [inline src/app/DataAccessLayer/dalMain.cs]: remove TODO

**Review by bartr — CHANGES_REQUESTED** [review]
comments in PR
dotnet test fails
build fails
need to run lint on code
nuget package updates

File: src/app/DataAccessLayer/dalMain.cs (hunk 3 above)
Comment (mag16) [reply src/app/DataAccessLayer/dalMain.cs]: missed that..thanks Bart.  Removed.

File: src/app/GlobalSuppressions.cs (hunk 1 above)
Comment (mag16) [reply src/app/GlobalSuppressions.cs]: Resolved.

**Review by aflinchb — APPROVED** [review]

**Review by bartr — APPROVED** [review]

**Pull request merged**